unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -283,5 +286,51 @@ private void UnEquipWearable(string wearableId)
 
             view.UpdateAvatarPreview(model.ToAvatarModel());
         }
+
+        private void ToggleSlot(string slotCategory, bool supportColor, bool isSelected)
+        {
+            currentSlotSelected = isSelected ? slotCategory : null;
+            view.SetColorPickerVisibility(isSelected && supportColor);
+
+            switch (slotCategory)
+            {
+                case "eyes":
+                    view.SetColorPickerValue(model.eyesColor);
+                    break;
+                case "hair" or "eyebrows" or "facial_hair":
+                    view.SetColorPickerValue(model.hairColor);
+                    break;
+                case "bodyshape":
```

Comment:
Let's make some constants of the categories

Reply:
Done!